### DISCUSSION TITLE
Can Moes Zigbee wired gateway be flashed?

### DISCUSSION THREAD

**Discussion by sagioto** [arendst/Tasmota#13873] [Q&A] [answered]
Hi,

I have this device: https://fr.aliexpress.com/item/4000465421911.html?spm=a2g0s.9042311.0.0.4fe06c37cr9kmI
Can it be flashed with Tasmota so I can use ZHA and use Aqare zigbee devices as well?

**Comment by ascillato** [comment, marked as answer]
As this device is not listed between the known supported ones https://zigbee.blakadder.com/search.html, we can't tell. If it has the supported Wi-Fi chips (ESP8266 or ESP32) and the supported zigbee chips, it should work. Otherwise won't work.
